src/dbos-executor.ts

```diff
@@ -156,7 +156,7 @@ export class DBOSExecutor implements DBOSExecutorContext {
   readonly logger: Logger;
   readonly tracer: Tracer;
   // eslint-disable-next-line @typescript-eslint/ban-types
-  entities: Function[] = [];
+  entities: Function[] | { [key: string]: object } = [];
```

Comment:
Thanks for cleaning up the decorators file!  Can you clean up this file as well?